src/main/java/org/cactoos/scalar/Reduced.java

```diff
@@ -65,6 +66,18 @@ public Reduced(final X ipt, final BiFunc<X, T, X> fnc,
         this.func = fnc;
     }
 
+    /**
+     * Secondary ctor.
```

Comment:
@izrik Just `Ctor.`. There are not line as `Secondary ctor` in the whole project.